csi/controllerserver.py

```diff
@@ -469,6 +469,20 @@ def ControllerExpandVolume(self, request, context):
             context.set_code(grpc.StatusCode.INVALID_ARGUMENT)
             return csi_pb2.ControllerExpandVolumeResponse()
 
+        if not request.capacity_range:
+            errmsg = "Capacity Range is empty and must be provided"
+            logging.error(errmsg)
+            context.set_details(errmsg)
+            context.set_code(grpc.StatusCode.INVALID_ARGUMENT)
+            return csi_pb2.ControllerExpandVolumeResponse()
+
+        if not request.capacity_range.required_bytes:
+            errmsg = "Capacity Range is empty and must be provided"
```

Comment:
Can this errmsg be tweaked not to sound same as above?

Reply:
Ack

Reply:
Addressed in #571